File: Banebooking.Api/Dtos/Booking/MassebookingDto.cs
namespace Banebooking.Api.Dtos.Booking;

public class MassebookingDto
{
    public DateOnly StartDato { get; set; }
    public DateOnly SluttDato { get; set; }

    public List<DayOfWeek> Ukedager { get; set; } = new();
    public List<TimeOnly> Tidspunkter { get; set; } = new();
    public List<Guid> BaneIder { get; set; } = new();

    public string Bookingtype { get; set; } = string.Empty;
    public string? Kommentar { get; set; }
}

File: Banebooking.Api/ClientApp/src/types/MassebookingDto.ts
export type MassebookingDto = {
    startDato: string;           // yyyy-MM-dd
    sluttDato: string;
    ukedager: string[];          // f.eks. ["Monday", "Wednesday"]
    tidspunkter: string[];       // "08:00"
    baneIder: string[];          // GUIDs
    bookingtype: string;         // "kurs"
    kommentar: string;
};
